wgsl/index.bs

```diff
@@ -1043,6 +1058,20 @@ array_decoration
   </xmp>
 </div>
 
+<div class='example' heading='Access qualifier'>
+  <xmp>
+    # Storage buffers
+    var<storage> buf1 : [[access(read)]] Buffer;       # Can read, cannot write.
+    var<storage> buf2 : [[access(read_write)]] Buffer; # Can both read and write
```

Comment:
Question, for clarity: 

Are      
             [[access(read)]] Buffer   
and
             [[access(read_write)]] Buffer

different types?    I expect they should be. For example, forming a pointer into them should permit different behaviours.

Reply:
if you say so :)